### DISCUSSION TITLE
中央大學是美食沙漠嗎

### DISCUSSION THREAD

**Discussion by 113607514** [1101-GS4719-C/Python-Community-GS4719-C#2226] [General]
每天生活在中央大學，想問各位同學吃的還習慣嗎？大一的我生活到現在每天都吃得很飽，不知道大二的我會不會吃膩。我覺得後門的飽咖咖是我的愛店之一，我最常點的是雙份雞胸肉套餐，平常健身完吃如此多的雞胸肉讓我感到非常滿足。除此之外，還有許多經濟實惠的餐廳例如：歐姆萊斯、迷路和一品鍋⋯還有許多餐廳我還沒吃過，等著我去挖掘。但中央大學號稱美食沙漠，我都吃的那麼開心了。很難想像其他大學的美食和餐廳有多好吃。

**Comment by yuan3x2** [comment]
我認為飽咖咖有一點衛生疑慮，但弟弟妹妹超可愛的，至於中央為什麼被稱為美食沙漠我猜主要是有時間、價格和交通上的加乘，畢竟要連續吃四年，學餐貴又難吃，外食也都不便宜，而且交通又不是特別方便，公車久久一班，假日又很多店都沒開，大概正是如此才有此稱吧，希望我能撐到畢業。

**Comment by gigitilto** [comment]
對一個素食者來說，我確實能感受到中央沒有很多可以吃的東西。大一的時候，我幾乎是餐餐吃全家，但也不是說全家就有比較多可以吃的，只是相對來說比較方便而且當時認為85折特別好。到了大一下，我開始天天去熱浪島拿便當，因為實在是吃膩全家的東西了，而他們家的便當菜色每天都不同，而且感覺也比較健康。直到現在，我還是只有去過這兩個地方解決我的三餐，即便馬爾波、人行道sidewalk也有素的，但實在是太貴了。

**Reply by forestdew507** [reply]
如果是鍋邊素的話如果不嫌麻煩可以走到任何店裡問問看，搞不好還會有其他選項XD 我在宵夜街緣杏有看過有人問老闆有沒有素食，也看過有點素烏龍~

**Comment by Jinyuu17** [comment]
吃了一年後，我都在宿舍自己煮飯了。推薦大同電鍋和電磁爐，可以滿足九成的飲食需求，價錢比學校附近的都便宜，但就要花時間採買跟料理。

**Comment by Tchang323** [comment]
在中央生活，跟中原或台大那些學校比起來，美食的選擇真的不多，但待久了也就習慣了，覺得能吃就好，像我一個禮拜大概會吃四到五次的宵夜屋跟燒臘XD

**Comment by forestdew507** [comment]
個人偏好有蔬菜的店XD 宵夜街的宵夜屋、25 食堂、翔食堂；後門的飽咖咖、米豆等 ( 突然要想只想到這幾間 )
FYI 如果在中央要吃水果可以沿著後門走到底右轉，那邊就有水果攤了 ! 另外如果是自己主要買菜，同一間水果攤也有賣菜噢:)

**Comment by lai1008** [comment]
中央大學真的沒什麼可以吃的，要好吃的就真的要走一大段路到後門或是宵夜街吃飯，所以推薦還是要有一台腳踏車才能在這個美食沙漠有很大的校園裡生存

**Comment by Boying12** [comment]
作為一名大四的學生，回想起我剛進中央大學的時候，對於校園裡的飲食還是充滿了新鮮感。那時候，我也像你一樣。
然而，隨著時間的推移，這種新鮮感漸漸消退了。每天面對同樣的選擇，逐漸讓我感到厭倦。雖然中央大學的餐廳數量並不少，但在這片號稱「美食沙漠」的地方，卻常常讓我感到無奈。每當吃到熟悉的菜色，心中那份曾經的期待也漸漸變成了失望。
現在，對於即將到來的每一餐，我總是懷著一種痛苦的心情，期待卻又害怕再度重複。大四的我，渴望逃離這樣的飲食循環，想要去探索其他大學的美食，去嘗試那些我未曾品嚐過的料理。想想其他校園裡的美味餐廳，讓我不禁想要走出這片熟悉的土地，尋找更豐富的味蕾體驗。

**Comment by yue083139** [comment]
感覺在中央大學的食物選擇上是真的不多，我基本上離開大一之後就沒有那種吃東西的新鮮感了，如果我想要吃一點好的，有時間的話倒是可以下山去市區，有些餐廳還是挺不錯的，只是就算我是騎摩托車還是感覺有點麻煩，更不用提學校的公車少的可憐。

**Comment by qiu0109** [comment]
中央附近以後門跟宵夜街來說的話，是真的沒什麼好吃的，一開始會覺得新鮮但到大一下基本上都會吃膩，但我想不管是在哪一個學校，如果只吃學校附近的東西的話應該都差不多容易吃膩，所以多下山拓展自己的美食圈比較實際。

**Comment by ShuliANN** [comment]
我非常認同樓上所說：其實，這種情況不只發生在中央，幾乎所有學校附近的飲食圈都是這樣。如果只侷限在學校附近用餐，難免會覺得單調。所以，與其抱怨沒有新鮮感，不如多下山或探索更遠的地方，將美食範圍擴展到更廣的區域，這樣不僅能嘗到更多元的美食，也是一種放鬆與生活調劑的方式。

**Comment by ShuliANN** [comment]
不過以我個人而言，比起下山還更常自己在宿舍煮，可以吃自己想吃的，也能確保衛生和均衡飲食！
宿舍的電鍋、微波爐好方便～！讀大學順便訓練廚藝～

**Comment by Elaine4488** [comment]
從大二開始，就常常在學校煩惱要吃什麼。最近看了不少簡單又容易上手的料理影片，決定開始自己動手煮飯，不僅能解決吃飯的煩惱，還能更容易攝取到足夠的蔬菜和蛋白質！

**Comment by kzx-creat** [comment]
大三開始外宿就有自己下廚的習慣，因為學校的都吃膩了，大多時候也懶得特別下山買。住宿舍下廚比較麻煩，還要擔心冰箱食物被幹走或找不到.....。

**Comment by wcyeee** [comment]
我覺得還好耶 目前還找得到三家左右願意一直吃的店家，我覺得就已經很好了！個人住的城市可能要想超級久才能真的擠出一兩間很值得吃的好吃店家。

**Comment by Mminniehuang** [comment]
我也是這學期開始自己煮東西來吃，去costco買一箱麵條和特價的玉米濃湯，再去生鮮超市或是雜貨店買菜、蛋、肉或是偶爾買一些配料，真的省錢又健康！不過唯一的缺點就是廚藝有限菜色很單一，寒假要好好學煮飯才行😅

**Comment by Taoyi-Chiang** [comment]
以前會覺得很煩躁，選擇很少。
現在就覺得算了，隨便吃。
去市區吃的時間成本真的好高，實在很難每天都這樣搞。

**Comment by lichilun** [comment]
我覺得以中央大學的食物以多樣性來說其實種類蠻多的，但山下的食物確實比較好吃(下山真的好麻煩），但中央大學的全家真的很不錯，可以打85折！！！

**Comment by SendaiHatsuki** [comment]
說真的，中央附近走路能到的店真的不好吃，而且還貴，少數我覺得還可以的就只有迷路和廚窗。平時都是去全家吃85折，或是麵包+飲料49元。想晚餐要吃什麼比期中期末考還難。

**Comment by a414414** [comment]
如果是中央周圍的話那還真是美食沙漠，如果沒有下山吃的話，都不知道下一餐到底要吃什麼，選擇少。但我覺得香城燒臘跟竹香都是好吃的便當店，CP值高，還有食間的優惠特餐（打拋豬飯+特大鮮奶茶）很爽。
